README.md

```diff
@@ -7,180 +7,211 @@ gollum -- A wiki built on top of Git
+Gollum is a simple wiki system built on top of Git. A Gollum Wiki is simply a git repository (either bare or regular) of a specific nature:
+* A Gollum repository's contents are human-editable, unless the repository is bare. Pages are unique text files which may be organized into directories any way you choose. Other content can also be included, for example images, PDFs and headers/footers for your pages.
+* Gollum pages:
+	* May be written in a variety of markups (see the [Installation](#installation) section).
+	* Can be edited with your favourite system editor or IDE (changes will be visible after committing) or with the built-in web interface.
+	* Can be displayed in all versions (commits).
 
-Gollum is a simple wiki system built on top of Git.
+Gollum can be launched either as a webserver (with the web interface) or in "console mode", where you can use a predefined API to query and manipulate the repository. For more information, see the [Running](#running) and [Configuration](#configuration) sections.
 
-Gollum wikis are simply Git repositories that adhere to a specific format.
-Gollum pages may be written in a variety of formats and can be edited in a
-number of ways depending on your needs. You can edit your wiki locally:
+For more information on Gollum's capabilities and pitfalls:
 
-* With your favorite text editor or IDE (changes will be visible after committing).
-* With the built-in web interface.
-* With the Gollum Ruby API.
-
-Gollum follows the rules of [Semantic Versioning](http://semver.org/) and uses
-[TomDoc](http://tomdoc.org/) for inline documentation.
+1. [Syntax/capability overview for pages](https://github.com/gollum/gollum/wiki).
+2. [Known limitations](https://github.com/gollum/gollum/wiki/Known-limitations).
+3. [Security overview](https://github.com/gollum/gollum/wiki/Security).
 
 ## SYSTEM REQUIREMENTS
 
-- Python 2.5+ (2.7.3 recommended)
-- Ruby 1.9.3+ (1.9.3 recommended) (JRuby compatible!)
-- Unix like operating system (OS X, Ubuntu, Debian, and more)
-- Will not work on Windows with the default [grit](https://github.com/github/grit) adapter, but might work via JRuby (please let us know!)
+| Operating System | Ruby           | Adapters           | Supported | 
+| ---------------- | -------------- | ------------------ | --------- |
+| Unix/Linux-like  | Ruby 1.9.3+    | all except [RJGit](https://github.com/repotag/rjgit) | yes |
+| Unix/Linux-like  | [JRuby](https://github.com/jruby/jruby) (1.9.3+ compatible) | [RJGit](https://github.com/repotag/rjgit) | yes |
+| Windows          | Ruby 1.9.3+    | none               | no        |
+| Windows          | [JRuby](https://github.com/jruby/jruby) (1.9.3+ compatible) | [RJGit](https://github.com/repotag/rjgit) | almost<sup>1</sup> |
 
-## SECURITY
+**Notes:**
 
-Don't enable `--custom-css`, `--custom-js` or `--mathjax-config` unless you trust every user who has the ability to edit the wiki.
```

Comment:
I think this warning should still be in the readme, maybe as a note to the relevant options in the table with command line options?